common/v2/features/Dashboard/Settings/Export/Downloader.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button, Typography } from '@mycrypto/ui';
+import translate from 'translations';
+
+import { makeBlob } from 'utils/blob';
+
+const FullWidthButton = styled(Button)`
```

Comment:
This is unviable. You have three definitions of a styled component to override a missing props for ui/Button.
Create `v2/components/Button` that wraps ui/Button and accepts a prop to toggle fullwidth